info.json

```diff
@@ -1,8 +1,8 @@
 {
-  "VERSION": "0.0.1",
+  "VERSION": "0.0.2",
   "FORMAT_VERSION": "1.1",
   "AUTHOR": "Achilleas Koutsou, Julia Sprenger",
-  "COPYRIGHT": "(c) 2018, German Neuroinformatics Node",
+  "COPYRIGHT": "(c) 2019, German Neuroinformatics Node",
```

Comment:
Is the year required and needs to be updated every year? Is there a simpler solution for this?